### ISSUE TITLE
Deadlock working with postgresql

### ISSUE THREAD

**Issue by mtali** [jazzband/django-silk#371]
I don't know how to repricate this byt it seams silk cause deadlock on my database

```
django.db.utils.OperationalError: deadlock detected
DETAIL:  Process 2348 waits for ShareLock on transaction 28337148; blocked by process 2347.
Process 2347 waits for ShareLock on transaction 28337155; blocked by process 2348.
HINT:  See server log for query details.
CONTEXT:  while deleting tuple (367,10) in relation "silk_response"
```

**Comment by nasirhjafri** [comment]
Closing as duplicate of #294
